docs/products/SmartBear MCP Server/zephyr-integration.md

```diff
@@ -20,3 +20,8 @@ The following environment variables configure the Zephyr integration:
 
 - **Purpose**: Retrieve projects available within your Zephyr account.
 - **Returns**: A list of projects along with their properties, including information about if they have Zephyr enabled or not.
+
+### Get Test Cycles
+
+- **Purpose**: Retrieve Test Cycles available within your Zephyr account.
```

Comment:
If I look to other *.md files from other products I can see that they have additional information that could be useful as `Use case:` or `Parameters:`.